test/test_sink.py

```diff
@@ -741,10 +740,6 @@ def testFrameValues(use_add_tile_args, tmp_path):
     }
     compare_metadata(dict(sink.getMetadata()), expected_metadata)
 
-    sink.write(output_file)
-    written = large_image_source_zarr.open(output_file)
-    compare_metadata(dict(written.getMetadata()), expected_metadata)
```

Comment:
Rather than getting rid of this part of the test, we could replace the `compare_metadata` line with
```
    assert written.metadata['IndexRange'] == expected_metadata['IndexRange']
    assert written.metadata['IndexStride']['IndexC'] == 1
```